bach/tests/functional/bach/test_df_setitem.py

```diff
@@ -10,14 +10,20 @@
 from bach import SeriesInt64, SeriesString, SeriesFloat64, SeriesDate, SeriesTimestamp, \
     SeriesTime, SeriesTimedelta, Series, \
     SeriesJsonb, SeriesBoolean
-from tests.functional.bach.test_data_and_utils import get_bt_with_test_data, assert_db_type, \
-    assert_equals_data, CITIES_INDEX_AND_COLUMNS, get_bt_with_railway_data
+from sql_models.util import is_postgres
+from tests.conftest import get_postgres_engine_dialect
+from tests.functional.bach.test_data_and_utils import get_bt_with_test_data, assert_postgres_type, \
+    assert_equals_data, CITIES_INDEX_AND_COLUMNS, get_bt_with_railway_data, get_df_with_test_data
 
 
-def check_set_const(constant, db_type: str, expected_series: Type[Series]):
-    bt = get_bt_with_test_data()
+def check_set_const(engine, constant, expected_series: Type[Series], pg_db_type: str):
```

Comment:
```suggestion
def check_set_const(engine, constant, expected_series: Type[Series], expected_pg_db_type: str):
```
Was a bit difficult to understand that param (initially I thought it was casting), had to go and see what  ```assert_postgres_type``` was doing.

Reply:
good point